docs/configurations/02_sql_configurations.md

```diff
@@ -16,6 +16,7 @@ license: |
 <!--begin-include-->
 |Key | Default | Description | Since
 |--- | ------- | ----------- | -----
+spark.clickhouse.fixedStringReadAs|binary|Read ClickHouse FixedString type as the specified Spark data type. Supported types: binary, string|0.8.0
```

Comment:
after second thought, the name should be `spark.clickhouse.read.fixedStringAs`, to make it style consistent.

Reply:
The reason I'm not naming it this way is because seems all configurations named `spark.clickhouse.read.xyz` are processed by `ReadOptions`, which is not applicable in this case. But yes, I agree that this naming convention would be more consistent.

Reply:
updated.